pkg/api/v1/conversion.go

```diff
@@ -0,0 +1,2810 @@
+/*
+Copyright 2015 Google Inc. All rights reserved.
```

Comment:
Why do we need all these conversions? I'd expect the internal rep to be very close to the v1 API.